hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AInputStream.java

```diff
@@ -146,9 +147,12 @@ public S3AInputStream(S3AReadOpContext ctx,
     this.serverSideEncryptionAlgorithm =
         s3Attributes.getServerSideEncryptionAlgorithm();
     this.serverSideEncryptionKey = s3Attributes.getServerSideEncryptionKey();
+    ChangeDetectionPolicy changeDetectionPolicy =
```

Comment:
just do this in the `new ChangeTracker()` call

Reply:
Updated in https://github.com/apache/hadoop/pull/675/commits/90d5c9c16277bed3deb7c6ff760d875c1123fe83